.github/workflows/e2e-tests.yml

```
@@ -623,6 +629,7 @@ jobs:
           MOCK_MODEL_SERVER_URL: http://localhost:9999
           E2E_LOCAL_CLAUDE_MODEL_SERVER_URL: http://localhost:9999
           E2E_DEVICE_ID: e2e-claudecode-device
+          E2E_SHARD_INDEX: ${{ matrix.shardIndex }}
```

actionlint (1.7.12)

[error] property "shardindex" is not defined in object type {}

(expression)